src/Files.App/Helpers/ContextFlyoutItemHelper.cs

```diff
@@ -387,20 +387,11 @@ private static bool Check(ContextMenuFlyoutItemViewModel item, CurrentInstanceVi
 					IsVisible = itemsSelected,
 					IsPrimary = true,
 				}.Build(),
-				new ContextMenuFlyoutItemViewModel()
+				new ContextMenuFlyoutItemViewModelBuilder(commands.ShowProperties)
 				{
-					Text = "Properties".GetLocalizedResource(),
 					IsPrimary = true,
-					OpacityIcon = new OpacityIconModel()
-					{
-						OpacityIconStyle = "ColorIconProperties",
-					},
-					Command = commandsViewModel.ShowPropertiesCommand,
-					ShowInRecycleBin = true,
-					ShowInSearchPage = true,
-					ShowInFtpPage = true,
-					ShowInZipPage = true,
-				},
+					IsVisible = commands.ShowProperties.IsExecutable && itemsSelected,
```

Comment:
Shouldn't `itemsSelected` be part of `IsExecutable`?

Reply:
Of course:)